docs/dl-sandbox.md

```diff
@@ -4,48 +4,49 @@ This document explains how to get started with Scalar DL Sandbox.
-We will give you a zip file containing an access token, a key pair and configuration file.
-The access token is used for communicating with Sandbox API gateway to authenticate you.
+We will authorize you through GitHub OAuth to grant you access to the Sandbox environment.
+Please visit [our sandbox site](https://scalar-labs.com/sandbox/), read the [terms of use](https://scalar-labs.com/terms-of-use), and press the button `Try Now`.
+We will provide you a zip file containing the necessary access token, key pair and configuration file.
+The access token is only used for autehnication with Sandbox API gateway.
 The key pair is used for communicating with Scalar DL network.
 
-Please note that we give you a generated key pair for ease of use in the Sandbox, but it is usually required to create your private key in your own environment.
+Please note that we generate a key pair for ease of use for the Sandbox environment, but it is usually required to create your private key in your own environment.
 
 ## Before running your first contract 
 
 Let's clone Scalar DL client SDK to interact with Scalar DL network.
 ```
 $ git clone https://github.com/scalar-labs/scalardl-client-sdk.git 
 ```
-You can put your downloaded zip in the directory and unzip it.
+Unzip the downloaded zip file into this directory.
 
 Scalar DL manages data as a set of assets, where each asset is composed of a history of data identified by a key called `asset_id` and historical version number called `age`.
 `asset_id` is an arbitrary, but unique, string specified by users to manage their assets.
 
-However, the Sandbox is a shared environment that anyone can access,
-so please be careful about choosing an appropriate name so that it will not conflict with `asset_id`s chosen by other users.
-One recommended way to do this is to append your username to your asset name as in `<username>-<your-asset-name>`.
-Also, the same care is needed in choosing contract IDs. So, we recommend using `<username>-<your-contract-class-name>`.
+Since the Sandbox is a shared environment that anyone can access,
+take speical care when choosing an appropriate name so that it will not conflict with the `asset_id` chosen by another user.
+One recommended way to do this is to append your username to the asset name, for example `<username>-<your-asset-name>`.
+Also, the same care is needed when choosing contract IDs and we recommend using `<username>-<your-contract-class-name>`.
 
 ## Register your certificate
 
 Next, let's register your certificate in the Scalar DL network.
-The registered certificate will allow you to register and execute contracts, and will also be used for tamper detection of the data stored in the network.
+The registered certificate will allow you to register and execute contracts, and it is used for tamper detection of the data stored in the network.
```

Comment:
I feel here like we need the "also" here. What do you think?

Reply:
So something like "and also used for ..."